lib/resque/scheduler.rb

```diff
@@ -189,12 +188,15 @@ def handle_delayed_items(at_time = nil)
               items.each do |item|
```

Comment:
GitHub doesn't let me comment on the actual line, but there's another `master?` check a few lines above this one (186/185). Shouldn't we remove that one too?

Reply:
I think that means your test is not testing the right thing? It's passing even though only one instance is pushing delayed jobs.

Reply:
I'm already sort of doing that but it's still passing even though it shouldn't 😕